### ISSUE TITLE
Card Prices

### ISSUE THREAD

**Issue by dekkerglen** [dekkerglen/CubeCobra#67]
Use TCGPlayer API to get card prices as a field on cards. I think the best way to do this is to add a function to add card prices to cube objects as they are retrieved, on the server side. Some cards won't have a tcgplayer ID, because tcgplayer doesn't sell them, so we will have to account for that. (don't make the cards free)

You will need a TCGPlayer API key, please put it, and reference it from, cubecobrasecrets. We can't keep those API keys on the repo (obviously).

**Comment by dekkerglen** [comment]
I have the Cube Cobra TCGplayer API key. I'll start development this weekend, will have it out in 1-2 weeks I suspect.

**Comment by SeanmanX** [comment]
Should add the fact that the tcgplayer api key is needed in `README.md`...

**Comment by dekkerglen** [comment]
Yes, I need to revise this feature as well so if no API key is present the application still works. I'll update the readme as well.
